### PR TITLE
fix: read data from bigquery table

### PR DESCRIPTION
**Pull request by tmatsuo**
We need the table name

### CODE REVIEW

File: src/issue.ts [hunk 1]
```diff
@@ -121,12 +122,105 @@ export async function getIssues(flags?: Flags): Promise<IssueResult[]> {
...
 }
 
+async function getRepoIssuesFromBigQuery(repo: Repo, flags?: Flags): Promise<IssueResult> {
+  const [owner, name] = repo.repo.split('/');
+  const result = {issues: new Array<Issue>(), repo};
+  const bigquery = new BigQuery();
+
+  const tableName = '';
+  const query = 'SELECT ' +
+    'assignees, closed, closedAt, createdAt, isPr, issueId, issueType, labels, ' +
```
Comment (jmdobry) [inline src/issue.ts:138]:
```sql
SELECT
  assignee_github_logins AS assignees,
  issue_closed AS closed,
  close_time AS closedAt,
  create_time AS createdAt,
  is_pr AS isPr,
  issue_id AS issueId,
  issue_type AS issueType,
  labels AS labels,
  priority AS priority,
  IF(priority = 'PRIORITY_UNSPECIFIED', TRUE, FALSE) AS priorityUnknown,
  repo_name AS repo,
  reporter_github_login AS reporter,
  title AS title,
  update_time AS updatedAt,
  CONCAT('https://github.com/', repo_name, '/', IF(is_pr, 'prs', 'issues'), '/', issue_id) AS url
FROM `devrel-public-datasets-prod.github.github_issues`
WHERE
  issue_closed = FALSE
  AND repo_name == "${repo.repo}";
```

**Review by jmdobry — CHANGES_REQUESTED** [review]

File: src/issue.ts (hunk 1 above)
Comment (tmatsuo) [reply src/issue.ts]: done

File: src/types.ts [hunk 2]
```diff
@@ -147,6 +147,20 @@ export interface IssuesApiResponse {
   nextPageToken: string;
 }
 
+export interface BigQueryIssue {
+  labels: string[];
```
Comment (jmdobry) [inline src/types.ts:151]: I believe this will come out of BigQuery as a comma-delimited string

File: src/types.ts [hunk 3]
```diff
@@ -147,6 +147,20 @@ export interface IssuesApiResponse {
...
+  isPr: boolean;
+  repo: string; // googleapis/nodejs-rcloadenv
+  createdAt: string;
+  updatedAt: string;
+  issueId: number;
+  title: string;
+  priority: string;
+  assignees: Array<string>;
```
Comment (jmdobry) [inline src/types.ts:159]: I believe this will come out of BigQuery as a comma-delimited string

**Review by jmdobry — CHANGES_REQUESTED** [review]

File: src/types.ts (hunk 2 above)
Comment (tmatsuo) [reply src/types.ts]: Done

**Comment by tmatsuo** [comment]
Ok, now it's a decent chance that the code will work. I'd like to test it with test spreadsheet.

**Review by jmdobry — APPROVED** [review]

**Review by bcoe — APPROVED** [review]

**Pull request merged**